modules/nw-nodeport-service-range-edit.adoc

```diff
@@ -8,6 +8,13 @@
 
 You can expand the node port range for the cluster.
 
+[IMPORTANT]
+====
+It is possible to expand the node port range into the protected port range, which is between 0 and 32767. After expansion, the range cannot be reduced. Attempting to reduce the change returns the following error: `The Network "cluster" is invalid: spec.serviceNodePortRange: Invalid value: "30000-32767": new service node port range 30000-32767 does not completely cover the previous range 0-32767`. 
+
+Before making changes, ensure that the new range you set is appropriate for your cluster.
```

Comment:
```suggestion
You can expand the node port range into the protected port range, which is between 0 and 32767. However, after expansion, you cannot change the range. Before making changes, ensure that the new range you set is appropriate for your cluster. Attempting to reduce the change returns the following error: 

[source,text]
----
The Network "cluster" is invalid: spec.serviceNodePortRange: Invalid value: "30000-32767": new service node port range 30000-32767 does not completely cover the previous range 0-32767
----
```

- Suggestion to consider an active voice. "you cannot change the range...."


- IMO "Before making changes, ..." would be better before mentioning the error. This and other suggested changes are up to you though.

Reply:
Made some of the suggested changes, but not all. Keeping the error text as-is because I don't want to put a source text box inside of an important admonition box.